### DISCUSSION TITLE
Implementation

### DISCUSSION THREAD

**Discussion by Pinnokkio** [creativelifeform/three-nebula#158] [General]
Hey,

I'm really interested in implementing this example:
https://three-nebula.org/examples/point-zone
https://github.com/creativelifeform/three-nebula/blob/master/website/components/Examples/SpriteRendererPointZone/init.js

But how do I do this? I'm not using React, Vue or similar. Just plain js.
Thanks in advance!

**Comment by rohan-deshpande** [comment]
The code in this example is pretty much pure JS... what issues are you having? Can you write a [codesandbox](https://codesandbox.io) so I can take a look at where you're getting stuck?
